torch/_inductor/decomposition.py

```diff
@@ -29,6 +29,7 @@
 inductor_decompositions = get_decompositions(
     [
         aten._adaptive_avg_pool2d_backward,
+        aten.addmm,
```

Comment:
I think this is causing the test failures.

Reply:
Yeah, quite a few accuracy issues. I remove this part from this PR. It may take more time to triage and fix those issues.